### ISSUE TITLE
[Question] Choosing efxclipse components

### ISSUE THREAD

**Issue by nlisker** [eclipse-efx/efxclipse-eclipse#63]
In the 4.11 release, the update site has
`e(fx)clipse - Minimal JavaFX OSGi integration bundles	3.5.0.201902220600`
with the description:
> Minimal feature used to integrate javafx using adapter hooks which is needed when embedding into SWT and/or you are not able to modify the default classloader delegation to ext

I'm not using SWT. What I need are the css tools. Are they included, and can I choose which components to install?

**Comment by tomsontom** [comment]
well the minimal feature is for components like eg GEF-5 who only want the SWT integration. If you want the CSS-Support you require the minimal feature (IIRC we use it to show eg the gradient-tool in the editor).

I would in general advise to install the complete suite. The CSS-Editor eg depends on Xtext, ... - what's your concern?

**Comment by nlisker** [comment]
Not much of a concern, it's just that the description makes it look like it's mainly focused on SWT and I wasn't sure if the css and fxml extensions are included.

**Comment by tomsontom** [comment]
hm where do you see that feature in https://download.eclipse.org/efxclipse/updates-released/3.5.0/site/? Please note e(fx)clipse dropped from the release train!

**Comment by tomsontom** [comment]
You need the feature found in "e(fx)clipse - install > e(fx)clipse IDE"!

**Comment by nlisker** [comment]
In the update site https://download.eclipse.org/releases/2019-03/.

**Comment by tomsontom** [comment]
From that update-site you don't get the e(fx)clipse tooling any more! The e(fx)clipse stuff you find in the sim-rel repo are only the ones for dragged in for GEF-5. 

The reason we dropped from the release train is that we managed to provide bits who when installed, potentially rendered Eclipse unusable for 2 releases (=it didn't start if you run on Java-11) hence we decided to drop from the train so that people don't blame the Eclipse IDE for a bad experience cause by a bogus plugin.

We simply don't have the man power (=funding) to work on the IDE tooling story. We are strictly paid for our runtime stuff which is a bad situation but nothing we can change.
